Dockerfile

```diff
@@ -0,0 +1,38 @@
+# syntax=docker/dockerfile:1
+FROM node:21.5
+
+ENV SKIP_MAKE_FILES=${SKIP_MAKE_FILES}
+
+COPY . /Cosmos-Keyboards/
```

Comment:
Wouldn't this `COPY` interfere with the bind mount you have set up in the `docker-compose`? Or does live-reloading the page when you edit a source file on your computer still work?

I wonder if the three first three lines of code in `entry-point.sh` could be avoided if you copy into a different folder (say `./Cosmos-Keyboards-Build`) and did the building there, then copy `./Cosmos-Keyboards-Build/target` to `./Cosmos-Keyboards/target` at the end of the build script.

Reply:
The volume is created only after the container is built. So during container creation it needs to be copied. Immediately after the container is built any files created in the volume will be wiped out. Copying all files from `/Cosmos-Keyboards` to `/temp-cosmos` and back means all local changes will be present, along with the newly created files during the build process. But this could be made more efficient by working in another directory in `Dockerfile`.